internal/uvm/create_wcow.go

```diff
@@ -35,9 +35,12 @@ type OptionsWCOW struct {
 // `owner` the owner of the compute system. If not passed will use the
 // executable files name.
 func NewDefaultOptionsWCOW(id, owner string) *OptionsWCOW {
-	return &OptionsWCOW{
+	opts := &OptionsWCOW{
 		Options: newDefaultOptions(id, owner),
 	}
+	// Enable external guest connection for all WCOW by default
+	opts.ExternalGuestConnection = true
```

Comment:
If we're going to enable this by default for LCOW and WCOW now, why not just set **ExternalGuestConnection** to true in **newDefaultOptions**?